### PR TITLE
fix: Ensure policy output dir

### PR DESCRIPTION
**Pull request by zagronitay**
🎉 Thank you for making CloudQuery awesome by submitting a PR 🎉

#### Summary

---

Use the following steps to ensure your PR is ready to be reviewed

- [ ] Read the [contribution guidelines](../blob/main/.github/CONTRIBUTING.md) 🧑‍🎓
- [ ] Run `go fmt` to format your code 🖊
- [ ] Lint your changes via `golangci-lint run --new-from-rev main` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation))
- [ ] Update or add tests 🧪
- [ ] Ensure the status checks below are successful ✅

### CODE REVIEW

**Review by spangenberg — APPROVED** [review]
LGTM

**Review by bbernays — APPROVED** [review]

File: pkg/policy/execute.go [hunk 1]
```diff
@@ -358,10 +358,11 @@ func GenerateExecutionResultFile(result *ExecutionResult, outputDir string) erro
 	fullPath := fmt.Sprintf("%s.json", filepath.Join(outputDir, result.PolicyName))
 
 	// Ensure entire directory structure is created
-	directory := fullPath[:strings.LastIndex(fullPath, "/")]
-
-	if err := fs.MkdirAll(directory, 0755); err != nil {
-		return err
+	if strings.LastIndex(fullPath, "/") > -1 {
+		directory := fullPath[:strings.LastIndex(fullPath, "/")]
```
Comment (irmatov) [inline pkg/policy/execute.go:362]: `if path.Dir(fullPath) != "." {
    if err := fs.MkdirAll(....
`

should that do the trick?

**Comment by shimonp21** [comment]
I suggest holding off on this change until monday when we can discuss it with @roneli .
In version `0.23.2` running policy `aws//cis_v1.2.0` created a file named `aws.json`, but since version `0.23.3` running policy `aws//cis_v1.2.0` will create a file `aws/cis_v1.2.0.json`. This can break customer's CIs, so I think we should roll back to the `0.23.2` behaviour (rather than fixing the `0.23.3` behaviour.

**Comment by shimonp21** [comment]
fixed in https://github.com/cloudquery/cloudquery/pull/770.

**Pull request closed without merging**